index.html

```diff
@@ -303,6 +301,50 @@ <h5 class="header-5">About Me</h5>
             </li>
         </ul>
     </section>
-</body>
-
+    <section id="contact-form">
+        <div id="header-6">
+            <h3>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
```

Comment:
- [ ] Please change font of the heading following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). Keep coding 💪